src/payments/certificate.controller.ts

```
@@ -0,0 +1,57 @@
+import {
+  Controller,
+  Post,
+  Body,
+  UsePipes,
+  ValidationPipe,
+  HttpCode,
+  HttpStatus,
+  UseFilters,
+} from '@nestjs/common';
```

ESLint

[error] Replace `'@nestjs/common'` with `"@nestjs/common"`

(prettier/prettier)